q2_annotate/plugin_setup.py

```diff
@@ -1323,19 +1322,19 @@
 plugin.methods.register_function(
     function=q2_annotate.kaiju._classify_kaiju,
     inputs={
-        "seqs": SampleData[
+        "reads": SampleData[
```

Comment:
Let's leave this at `seqs` - we have an open issue to introdcue more input types.